R/shinywrappers.R

```diff
@@ -271,15 +274,50 @@ markOutputAttrs <- function(renderFunc, snapshotExclude = NULL,
 #' shinyApp(ui, server)
 #' }
 renderImage <- function(expr, env=parent.frame(), quoted=FALSE,
-                        deleteFile=TRUE, outputArgs=list()) {
+                        deleteFile, outputArgs=list()) {
   installExprFunction(expr, "func", env, quoted)
 
+  # missing() must be used directly within the function with the given arg
+  if (missing(deleteFile)) {
```

Comment:
Why not set `deleteFile = NULL` in the function def?

Reply:
So you could look at the signature and know that `deleteFile` is "required". (Even though it's not truly required right now, because we're being nice and giving you a warning instead of an error.)

Reply:
Oh yeah, makes sense.

Reply:
Since our end goal is for people to always use either `TRUE` or `FALSE`, providing `NULL` as a default might make give people the idea that it's an OK value to use (even if only temporarily).